pkg/build/builder/source.go

```diff
@@ -252,7 +252,17 @@ func extractGitSource(gitClient GitClient, gitSource *api.GitBuildSource, revisi
 		}
 
 		if err := gitClient.Checkout(dir, commit); err != nil {
-			return true, err
+			glog.V(4).Infof("Checkout after clone failed for ref %s with error: %#v, attempting fetch", commit, err)
```

Comment:
just making sure @bparees - you decided you were OK with this and you posted the merge comment ?